MaterialDesignThemes.Wpf/GridViewColumnThumb.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Windows.Controls.Primitives;
+using System.Windows.Input;
+
+namespace MaterialDesignThemes.Wpf
+{
+    public class GridViewColumnThumb : Thumb
```

Comment:
For now lets make this an internal class. I am trying to avoid accidently exposing a class that we may change in the future.

Reply:
Done